### DISCUSSION TITLE
Timeline for inputcapture being merged into main

### DISCUSSION THREAD

**Discussion by benmordecai** [input-leap/input-leap#1935] [Q&A] [answered]
The [Archwiki page for inpuit-leap](https://wiki.archlinux.org/title/Input_Leap) is a mess because it is a mix of instructions for Synergy, Barrier, and Input-Leap and there is a proposal on the Talk page to split it off into its own page. 

If we do make a new page for Input-Leap on the Archwiki, I would like to have some clarity on that page for how to get Wayland support. Right now we install input-leap-inputcapture (AUR) to get Wayland support, but I am assuming in the long run this will be merged into input-leap-git (AUR). If this is going to happen shortly there is no point in writing out the whole process only for it to become immediately outdated when inputcapture gets merged. But if this is going to be a while, it would be worth writing out.

**Comment by shymega** [comment, marked as answer]
This isn't really our problem - it's up to each distro how they want to package Input Leap, and that includes the CMake option for libei (otherwise known as Input Capture). 

However, we do keep it disabled by default, because the portal interface isn't available on every distro right now. This is due to backporting and packaging processes.

If the Arch maintainers feel they can enable `libei`, then they can. But this is up to packagers.

Thanks.

**Reply by benmordecai** [reply]
Right now input leap is only in the AUR so it is not maintained by Arch packagers at all.

**Reply by shymega** [reply]
Yes, I understand - I was referring loosely to AUR/official packages. Again, this is up to the packagers to decide.

**Reply by sithlord48** [reply]
According to the arch wiki, The correct way to get input-leap into arch proper is for more ppl to vote for it 
https://wiki.archlinux.org/title/Arch_User_Repository#Voting_for_packages once we have an `input-leap` Non `-git` package it needs votes to be picked up by a maintainer.
